<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWin</class>
 <widget class="QWidget" name="MainWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>题型选择</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QGroupBox" name="gbxOpType">
        <property name="title">
         <string>运算类型选择</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QRadioButton" name="rabAdd">
           <property name="text">
            <string>加法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rabSub">
           <property name="text">
            <string>减法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rabMult">
           <property name="text">
            <string>乘法</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rabDiv">
           <property name="text">
            <string>除法</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxOpStep">
        <property name="title">
         <string>几步运算</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QRadioButton" name="rabOneStep">
           <property name="text">
            <string>一步</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rabTwoStep">
           <property name="text">
            <string>二步</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="ranTreeStep">
           <property name="text">
            <string>三步</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gbxSubjectType">
        <property name="title">
         <string>题目类型</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="rabResult">
           <property name="text">
            <string>求结果</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="rabItem">
           <property name="text">
            <string>求算数项</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_5">
     <property name="title">
      <string>详细设置</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string>运算项结果符合设置</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="btnOpResSet">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>运算项及结果范围设置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnOperatorSet">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>运算符号设置</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="ckbBracket">
        <property name="text">
         <string>使用括号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_11">
     <item>
      <widget class="QGroupBox" name="gbxAddSet">
       <property name="title">
        <string>加法设置</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QRadioButton" name="rabRandCarry">
          <property name="text">
           <string>随机进位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rabAddCarry">
          <property name="text">
           <string>加法进位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rabNoCarry">
          <property name="text">
           <string>没有进位</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbxSubSet">
       <property name="title">
        <string>减法设置</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QRadioButton" name="rabRandAbdicate">
          <property name="text">
           <string>随机退位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rabSubAbdicate">
          <property name="text">
           <string>减法退位</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="rabNoAbdicate">
          <property name="text">
           <string>没有退位</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="title">
      <string>添加计算题到试卷</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>题目个数：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spbSubjectCount">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>999</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnAddSubject">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>添加题目</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClearSubject">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>清空题目</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_10">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>200</height>
      </size>
     </property>
     <property name="title">
      <string>当前试卷包含题目：</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QTextBrowser" name="txbContent"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_11">
     <property name="title">
      <string>试卷设置</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="5" colspan="3">
       <widget class="QLineEdit" name="lneSubTitle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QSpinBox" name="spbColCount">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>5</number>
        </property>
       </widget>
      </item>
      <item row="0" column="7">
       <widget class="QPushButton" name="btnBrowse">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSpinBox" name="spbTotal">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>试卷列量：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="6">
       <widget class="QLineEdit" name="lneOutPath"/>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="lneTitle">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>生成试卷数量：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>试卷副标题：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>输出目录：</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>试卷标题：</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnGen">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>35</height>
      </size>
     </property>
     <property name="text">
      <string>开始生成试卷</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
